tests/vectors/main.rs

```diff
@@ -28,7 +46,7 @@ mod cap26_tests {
     #[allow(dead_code)]
     #[derive(Deserialize, Debug)]
     struct CAP26Vector {
-        path: HDPath,
+        path: CAP26Path,
```

Comment:
I've hidden more low level derive public key method, and since this is an integration tests we cannot use `pub(crate)` visibility... so we upgrade to a `CAP26Path` which is the type of the public method for deriving, but to support the theoreyical (not possible in wallets) test vectors of CAP26 over `Secp256k1` - we need the `TestDerivation` struct declared above.